### PR TITLE
feat(wren-ui/api): save columns when creating view

### PR DESCRIPTION
**Pull request by wwwy3y3**
## Description
* Add `columns` in view properties
* Add `fields` to view field in diagram API

### View field in diagram API
When querying
```graphql
query Diagram {
  diagram {
    views {
      displayName
      id
      nodeType
      referenceName
      statement
      viewId
      fields {
        displayName
        id
        nodeType
        referenceName
        type
      }
    }
  }
}
```

You'll get following response for `views`
```js
// ...
views: [
{
          "displayName": "view_3",
          "id": "c80d524c-a0a7-4d0a-a118-a6e63e0b41c8",
          "nodeType": "VIEW",
          "referenceName": "view_3",
          "statement": "-- Selecting specific columns from the customer table\nSELECT\n  c_custkey,\n  c_name,\n  c_address,\n  c_phone,\n  c_acctbal,\n  c_mktsegment,\n  c_comment\nFROM\n  customer",
          "viewId": 4,
          "fields": [
            {
              "displayName": "c_custkey",
              "id": "dae76223-d5a2-4898-95a8-c06bbbf3b550",
              "nodeType": "FIELD",
              "referenceName": "c_custkey",
              "type": "int8"
            },
            {
              "displayName": "c_name",
              "id": "26e09286-39bc-4315-9f11-6fddc6c7daa0",
              "nodeType": "FIELD",
              "referenceName": "c_name",
              "type": "varchar"
            },
            {
              "displayName": "c_address",
              "id": "4872a9fd-1c2f-4533-a409-b6c08e0315dd",
              "nodeType": "FIELD",
              "referenceName": "c_address",
              "type": "varchar"
            },
            {
              "displayName": "c_phone",
              "id": "242cb2da-28ff-4817-b4ca-360253ee4a1f",
              "nodeType": "FIELD",
              "referenceName": "c_phone",
              "type": "varchar"
            },
            {
              "displayName": "c_acctbal",
              "id": "4719be88-0c4f-4f94-b385-fea3051e6a04",
              "nodeType": "FIELD",
              "referenceName": "c_acctbal",
              "type": "float8"
            },
            {
              "displayName": "c_mktsegment",
              "id": "ef18b0e4-e0b2-43e8-85d4-b34fb1c7cf62",
              "nodeType": "FIELD",
              "referenceName": "c_mktsegment",
              "type": "varchar"
            },
            {
              "displayName": "c_comment",
              "id": "23fb2a51-9ddd-433e-8d62-5471d2e3211c",
              "nodeType": "FIELD",
              "referenceName": "c_comment",
              "type": "varchar"
            }
          ]
        }
]
// ...
```

### CODE REVIEW

**Review by onlyjackfrost — APPROVED** [review]
LGTM

File: wren-ui/src/apollo/server/adaptors/wrenEngineAdaptor.ts [hunk 1]
```diff
@@ -185,6 +190,19 @@ export class WrenEngineAdaptor implements IWrenEngineAdaptor {
     }
   }
 
+  public async desribeStatement(
```
Comment (andreashimin) [inline wren-ui/src/apollo/server/adaptors/wrenEngineAdaptor.ts:193]:
```suggestion
  public async describeStatement(
```

File: wren-ui/src/apollo/server/adaptors/wrenEngineAdaptor.ts [hunk 2]
```diff
@@ -42,13 +42,18 @@ export interface QueryResponse {
...
+}
+
 export interface IWrenEngineAdaptor {
   deploy(deployData: deployData): Promise<DeployResponse>;
   initDatabase(sql: string): Promise<void>;
   putSessionProps(props: Record<string, any>): Promise<void>;
   queryDuckdb(sql: string): Promise<QueryResponse>;
   patchConfig(config: Record<string, any>): Promise<void>;
   previewData(sql: string, limit?: number): Promise<QueryResponse>;
+  desribeStatement(sql: string): Promise<DescribeStatementResponse>;
```
Comment (andreashimin) [inline wren-ui/src/apollo/server/adaptors/wrenEngineAdaptor.ts:56]:
```suggestion
  describeStatement(sql: string): Promise<DescribeStatementResponse>;
```

**Review by andreashimin — APPROVED** [review]
Besides a wrong typo, others LGTM

File: wren-ui/src/apollo/server/adaptors/wrenEngineAdaptor.ts (hunk 1 above)
Comment (wwwy3y3) [reply wren-ui/src/apollo/server/adaptors/wrenEngineAdaptor.ts]: addressed at bc077a5

File: wren-ui/src/apollo/server/adaptors/wrenEngineAdaptor.ts (hunk 2 above)
Comment (wwwy3y3) [reply wren-ui/src/apollo/server/adaptors/wrenEngineAdaptor.ts]: addressed at bc077a5

**Pull request merged**